docs/openapi/swagger.json

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
Do we need both json or yaml? Can we keep one or the other if possible? I might prefer yaml because json doesn't support comment and yaml is neat.

Reply:
Sure. However the JSON file will always be re-generated by Swagger. But I tried locally, it's okay to remove the JSON, so maybe we can just add it in the git ignore?